avocado/utils/iso9660.py

```diff
@@ -100,20 +102,20 @@ def can_mount():
     :rtype: bool
     """
     if not process.can_sudo():
-        logging.debug('Can not use mount: current user is not "root" and '
-                      "sudo is not configured.")
+        LOG.debug('Can not use mount: current user is not "root" and'
```

Comment:
This is removing the recently added  (and needed) white space at the end of the line.